src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj

```
@@ -5,9 +5,9 @@
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <VersionPrefix>2.0.2.0</VersionPrefix>
+    <VersionPrefix>2.0.3.0</VersionPrefix>
```

While this is not a breaking change, it's still a significant change, so let's increment minor version - `2.1.0.0`
```suggestion
    <VersionPrefix>2.1.0.0</VersionPrefix>
```